### PR TITLE
CI: Always store the new build-id

### PR DESCRIPTION
**Pull request by didrocks**
Separate upload to the Store from storing the new version so that even manual build can be uploaded to the Store.

UDENG-2387

### CODE REVIEW

File: .github/workflows/build-wsl.yaml [hunk 1]
```diff
@@ -213,35 +216,41 @@ jobs:
...
+          echo "::set-output name=has-changes::true"
+
+          if [ ${{ matrix.Upload }} != "yes" ]; then
+            echo "::notice::${{ matrix.AppID }} build ${build_id} ready for sideload or manual submission to the Microsoft Store"
+            exit 0
+          fi
 
-          if [ "${build_id}" = "0" ]; then
+          # If we are doing a first build for this distro, we don’t want to submit it to the store as first submission
+          # is manual.We let the other steps and jobs happening to save artifacts (build-id, fingerprints) of
```
Comment (CarlosNihelton) [inline .github/workflows/build-wsl.yaml:234]: nitpick:
```suggestion
          # is manual. We let the other steps and jobs happening to save artifacts (build-id, fingerprints) of
```

File: .github/workflows/build-wsl.yaml [hunk 2]
```diff
@@ -213,35 +216,41 @@ jobs:
...
+
           hasChanges="$(git diff --staged ${fingerprint_file})"
           cd -
-
           if [ -z "${hasChanges}" ]; then
             exit 0
           fi
 
-          echo "::set-output name=save-artifacts::true"
+          echo "::set-output name=has-changes::true"
```
Comment (CarlosNihelton) [inline .github/workflows/build-wsl.yaml:226]: We should set some point to stop using `set-output` and prefer `$GITHUB_OUTPUT`, but that can be set for later.

**Review by CarlosNihelton — APPROVED** [review]
All good to me. Just two minor nitpicks which are not a priority.

File: .github/workflows/build-wsl.yaml (hunk 2 above)
Comment (didrocks) [reply .github/workflows/build-wsl.yaml:226]: Agreed but for later :)

File: .github/workflows/build-wsl.yaml (hunk 1 above)
Comment (didrocks) [reply .github/workflows/build-wsl.yaml]: Thanks!

**Pull request merged**